docs/sources/features/datasources/x-ray.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "AWS X-Ray"
+description = "Guide for using X-Ray in Grafana"
+keywords = ["grafana", "x-ray", "guide"]
+type = "docs"
+aliases = ["/docs/grafana/latest/datasources/x-ray"]
+[menu.docs]
+name = "AWS X-Ray"
+identifier = "x-ray"
+parent = "datasources"
+weight = 5
++++
+
+# Using AWS X-Ray in Grafana
+
+Grafana ships with built-in support for X-Ray. Add it as a data source, then you are ready to
```

Comment:
Yeah not actually true :). We will actually need to move this into the x-ray plugin this should not be here in the main grafana documentation but as the plugin readme which is then used in the plugins market place for documentation.